app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -28,14 +28,20 @@
     <% if @step_by_step_page.has_draft? %>
       <div class="govuk-panel panel-default">
         <div class="panel-body">
-          <p class="govuk-body">Before publishing check for broken links.</p>
-          <p class="govuk-body">After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
-          <%= render "govuk_publishing_components/components/button", {
-            text: "Publish changes",
-            href: step_by_step_page_publish_path(@step_by_step_page)
-          } %>
+          <% if !@step_by_step_page.scheduled_for_publishing? %>
```

Comment:
Looking at this now, I wonder if it might be clearer to instead have methods like: 

* `@step_by_step_page.can_publish?` or 
* `@step_by_step_page.can_unpublish?`
* `@step_by_step_page.can_delete?`

etc, and wrap each action in one of each?

N.B. method names are only an example.

Reply:
If you do try this, I'd recommend creating these methods first (with unit tests!) and checking that the existing tests still pass, before adding the `scheduled_for_publishing?` checks.